framework/src/application.ts

```diff
@@ -126,19 +122,25 @@ export class Application {
 		}
 		this.config = mergedConfig;
 
-		// Private members
-		this._plugins = {};
 		// Initialize node
 		const { plugins, ...rootConfigs } = this.config;
 		this._node = new Node({
 			options: rootConfigs,
 		});
+		this._controller = new Controller({
+			appConfig: rootConfigs,
+			pluginConfigs: plugins,
+		});
 	}
 
 	public get networkIdentifier(): Buffer {
 		return this._node.networkIdentifier;
 	}
 
+	public get channel(): InMemoryChannel {
```

Comment:
before `app.run` if user accesses `app.channel` will it throw error? or should return be `InMemory | undefined`